app/src/main/java/com/kks/nimblesurveyjetpackcompose/ui/presentation/survey/SurveyDetailScreen.kt

```diff
@@ -145,14 +157,15 @@ fun SurveyDetailScreen(
                         pagerState.animateScrollToPage(currentPage + 1)
                     }
                 } else {
-                    // TODO: Implement submit survey action
+                    viewModel.submitSurvey(surveyId = survey.id)
```

Comment:
So this statement would preferably be 
```kotlin
else if (isLastPage) {
    viewModel.submitSurvey(survey.id)
}

Reply:
If I change like this I won't need the check which said before. I will change accordingly 👍